alerts_summary.sh

```diff
@@ -6,7 +6,7 @@ dependency_tree_summary () {
         cat dependency-tree.txt
     fi
     {
-        echo "### $INPUT_DIRECTORY$2"
+        echo "### $INPUT_DIRECTORY$i"
```

Comment:
I don't speak bash very well :) What's `$i`?

Reply:
`$i` is a variable in the array in the line [L95](https://github.com/pitch-io/clojure-dependabot/blob/db9605e2825e0f3a539a62a9f73f5d783ee5b309/alerts_summary.sh#L95), but I am testing if `$1` works, because it would be a better name for the variable, avoiding future troubles :)